src/modules/ce/components/admin/AdminCoordinatedEntry.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import AdminReferralsTable from './AdminReferralsTable';
+import CommonTabs from '@/components/elements/CommonTabs';
+import PageTitle from '@/components/layout/PageTitle';
+import AdminOpportunitiesTable from '@/modules/ce/components/admin/AdminOpportunitiesTable';
+
+const AdminCoordinatedEntry: React.FC = () => {
+  return (
+    <>
+      <PageTitle title='Coordinated Entry' overlineText='Admin' />
+      <CommonTabs
+        ariaLabel={'Admin CE Tabs'}
+        tabDefinitions={[
+          {
+            title: 'Available Units',
+            key: 'opportunities',
+            contents: <AdminOpportunitiesTable />,
+          },
+          {
+            title: 'Referrals',
```

Comment:
Just noting I've diverged from the designs to call this simply "Referrals" instead of "Ongoing referrals". The tab shows ongoing referrals by default, but the user can filter to see referrals with other statuses. Let me know if you think it looks okay.